app.js

```diff
@@ -21,10 +23,16 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cleanStrings);
-
+app.use(session({
+  secret: 's3cr3t',
+  resave: true,
+  saveUninitialized: true
+}));
+app.use(passport.initialize());
+app.use(passport.session());
 router(app);
 
-app.get('*', (request, response) => {
+app.get('/', (request, response) => {
```

Comment:
Why this?

Reply:
I figured that the * picks invalid routes and renders the welcome message. I'll create another function to handle both cases.